go/cmd/vtctldclient/command/workflows.go

```diff
@@ -35,11 +40,49 @@ var (
 		Args:                  cobra.ExactArgs(1),
 		RunE:                  commandGetWorkflows,
 	}
+
+	getWorkflowsOptions = struct {
```

Comment:
nit: structurally, we've been keeping the options structs directly above the command func that uses them